### PR TITLE
Enhance pull requests and issues using templates

### LINKED ISSUE #87: Github PR Template
It is quite useful to make sure to follow a standard process for every pull request and to have a to-do list for the author to check before requesting a review.

- Issues
- Pull requests
- [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
  - [Approval Groups](https://stackoverflow.com/questions/45806336/github-group-review)

### PR DESCRIPTION
**Pull request by jinmayamashita**
## Why

Closed #87
I want to simplify PR/Issue creation and review requests.

## How

- [x] To use a `CODEOWNERS` file
- [x] Creating a pull request template
- [x] Creating a issue template

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# global owners will be requested for review when someone opens a pull request.
+
+- @monstar-lab-oss/reactjs-boilerplate-maintainers
```
Comment (jinmayamashita) [inline .github/CODEOWNERS:3]: team `reactjs-boilerplate-maintainers` has been generated.
https://github.com/orgs/monstar-lab-oss/teams/reactjs-boilerplate-maintainers

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<!--- Provide a detailed description of the change or addition you are proposing -->
+
+## Context
+
+<!--- Why is this change important to you? How would you use it? -->
+<!--- How can it benefit other users? -->
+
+## Possible Implementation
+
+<!--- Not obligatory, but suggest an idea for implementing addition or change -->
```
Comment (jinmayamashita) [inline .github/ISSUE_TEMPLATE.md:14]: It seems possible to display various menus in a [YAML file](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) as follows. First of all, I added it simply. We will consider it when it becomes necessary.

スクリーンショット 2022-08-25 15 58 23

**Comment by hhofner** [comment]
LGTM, should merge to get some more traction on the repo.

**Review by G-N555 — APPROVED** [review]
thank you for enhancing our repository.

**Pull request merged**